tests/tcp-server/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.16-alpine as builder
```

Comment:
can we remove docker and go files now that image is available in docker? or move them inside data/? @lucacome @pleshakov what would be a better approach?

Reply:
I'd prefer to leave it. It doesn't interact with the rest of the code or require maintenance, but it will really help the next person that tries to change the tcp-server

Reply:
agree with Sean, better to leave it. If we have too many apps like that, we can put them in one folder like `test-apps` or something like that